Lib/_colorize.py

```diff
@@ -0,0 +1,45 @@
+import io
+import os
+import sys
+
+COLORIZE = True
+
+
+class ANSIColors:
+    BOLD_GREEN = "\x1b[1;32m"
+    BOLD_MAGENTA = "\x1b[1;35m"
+    BOLD_RED = "\x1b[1;31m"
+    GREEN = "\x1b[32m"
+    GREY = "\x1b[90m"
+    MAGENTA = "\x1b[35m"
+    RED = "\x1b[31m"
+    RESET = "\x1b[0m"
+    YELLOW = "\x1b[33m"
+
```

Comment:
```suggestion
ANSIColorsStub = type('ANSIColorsStub', (ANSIColors,), {x: "" for x in dir(ANSIColors) if not x.startswith('__')})

def select_colorization_class() -> ANSIColors:
  return ANSIColors if can_colorize() else ANSIColorsStub
```

Using `get_ansi_colors()` preserves IDE suggestions (ctrl+space) and centralizes, as well as abstracts color availability logic, improving code's for clarity and flexibility.